ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -259,6 +259,11 @@ post_provision_config_nodes() {
     # Reserve port ranges 31416-31516 for DAOS and CART servers
     echo 31416-31516 > /proc/sys/net/ipv4/ip_local_reserved_ports
 
+    # Remove DAOS dependencies to prevent masking packaging bugs
+    if rpm -qa | grep fuse3; then
```

Comment:
this would also match libfuse3 - is that intentional?

Reply:
Based on the `dnf -y erase fuse3\*` below, maybe this should be `grep '^fuse3'`?

Reply:
> this would also match libfuse3 - is that intentional?

It is intentional.  The whole point is to remove all \*fuse3\* packages so that the installation of daos-\* causes all necessary \*fuse3\* packages to be installed as dependencies.

Reply:
Arguably.  But this PR was always meant as a stop-gap (or belt and suspenders if you want to look at it that way) and the more proper and long term fix is to ensure these packages are not being installed in the test images.  I have updates to them on my plate already so can incorporate that fix also.

But one can also argue that `libfuse3` not being installed is a distro problem, not a DAOS one since if `libfuse3` is needed, then the `Requires: /usr/bin/fusermount3` should install it.  So ensuring that it's removed is not at all as important as making user `/usr/bin/fusermount3` is not installed.